style.css

```diff
@@ -1,7 +1,125 @@
-html {
-  font-size: 10px;
-}
-title {
-  background-color: #3E3C3C;
-  color: blue;
+@media only screen and (max-width: 1500px) {
+  .wrapper {
+    margin: 0 auto;
+    background-color: #1c1a19;
+    background-image: url(./images/Illustration.png);
+    background-repeat: no-repeat;
+    background-position: right bottom;
+    display: flex;
+    flex-direction: column;
+    width: 375px;
+    height: 600px;
+  }
+
+  .container {
+    background-color: #3e3c3c;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    padding: 1em;
+  }
+
+  .hour {
+    font-family: 'Roboto', Verdana, Geneva, Tahoma, sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    letter-spacing: -0.28px;
+    color: white;
+    flex: 1;
+  }
+
+  .indicators {
+    flex: 1;
+    align-self: center;
+    text-align: center;
+  }
+
+  .nav {
+    background-color: #3e3c3c;
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+  }
```

Comment:
- Kindly make sure that your project sticks as much as possible to the Figma template provided. The hamburger menu should be aligned to the right of the container. :eyes: 
- Please reduce the spaces between the lines of text to stick to the Figma template provided.
![image](https://user-images.githubusercontent.com/98363075/192418191-8c958819-9301-46f0-9203-5c13bc445b7b.png)